### PR TITLE
coala org change

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack aec81d6ef9d0093668389bc58bb3e25624cba534

**Comment by Adrianzatreanu** [comment]
ack a1a216c57a560c29f8739373e73e058baa9639f8

**Comment by Adrianzatreanu** [comment]
ack e492045487ae8026980a0523bfa9afb2fb21bf2e

**Comment by Adrianzatreanu** [comment]
ack c8fdce0c8d0edd34f08fdbde190c71c4ca070e45

File: README.rst [hunk 1]
```diff
@@ -189,15 +189,15 @@ License
...
 .. |Documentation Status| image:: https://readthedocs.org/projects/coala/badge/?version=latest
    :target: http://coala.rtfd.org/
-.. |codecov.io| image:: https://img.shields.io/codecov/c/github/coala-analyzer/coala-bears/master.svg?label=branch%20coverage
-   :target: https://codecov.io/github/coala-analyzer/coala-bears
+.. |codecov.io| image:: https://img.shields.io/codecov/c/github/coala/coala-bears/master.svg?label=branch%20coverage
+   :target: https://codecov.io/github/coala/coala-bears
 .. |gitter| image:: https://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-brightgreen.svg
-   :target: https://gitter.im/coala-analyzer/coala
-.. |AGPL| image:: https://img.shields.io/github/license/coala-analyzer/coala-bears.svg
+   :target: https://gitter.im/coala/coala
```
Comment (Adrianzatreanu) [inline README.rst]: coala.io/chat

**Comment by Adrianzatreanu** [comment]
unack aec81d6ef9d0093668389bc58bb3e25624cba534

**Comment by sils1297** [comment]
Be sure to check all links

**Comment by adtac** [comment]
reack https://github.com/coala/coala-bears/pull/830/commits/5babdc947909d87a6aaf7fcb93d883e758909799 https://github.com/coala/coala-bears/pull/830/commits/6a14212bc7d3d4b338bda0bd4628174c65a45070 https://github.com/coala/coala-bears/pull/830/commits/0b8eb4abe243411768774d15a188ff02a046f08d

File: README.rst (hunk 1 above)
Comment (adtac) [inline README.rst]: Now that the redirects work, we'll go with the gitter direct link

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/830/commits/274890a480f52a622fa173e2e099a75283043602

**Comment by jayvdb** [comment]
@rultor merge

**Comment by adtac** [comment]
reack https://github.com/coala/coala-bears/pull/830/commits/1f9addd9ec5b553a3566ef0ef0872ca7ad286a9b https://github.com/coala/coala-bears/pull/830/commits/5f20208ec8735fa3fc93e0c8b5e9ec732c992bd1 https://github.com/coala/coala-bears/pull/830/commits/b736b68392b0db97882ff3774e9b2f4848685890 https://github.com/coala/coala-bears/pull/830/commits/e0f0d794052e6104a8b5867b252f5f632c10b053

**Comment by jayvdb** [comment]
@rultor merge

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**